packages/logger/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@loopback/logger",
+  "version": "1.0.0",
+  "description": "A LoopBack component for logger support.",
+  "scripts": {
+    "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
+    "build": "npm run build:lib && npm run build:lib6",
+    "build:lib": "node ../../bin/compile-package es2017",
+    "build:lib6": "node ../../bin/compile-package es2015",
+    "clean": "rm -rf loopback-logger*.tgz lib* package",
+    "integration": "mocha --opts ../../test/mocha.opts 'test/integration/**/*.ts'",
+    "prepare": "npm run build",
+    "pretest": "npm run build",
+    "test": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts' 'test/integration/**/*.ts' 'test/acceptance/**/*.ts'",
+    "unit": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts'",
+    "verify": "npm pack && tar xf loopback-logger*.tgz && tree package && npm run clean"
+  },
+  "author": "IBM",
+  "license": "MIT",
+  "dependencies": {
+    "@loopback/context": "^4.0.0-alpha.9",
+    "@loopback/core": "^4.0.0-alpha.9",
+    "@types/pino": "^3.0.1",
+    "@types/shot": "^3.4.0",
+    "@types/sinon": "^2.3.3",
+    "@types/supertest": "^2.0.2",
```

Comment:
shot, sinon, supertest in regular (production) dependencies - that does not look right to me. Could you please review the dependencies and remove all unused ones?